src/conventions/h-group/clue-interpretation/interpret-clue.js

```diff
@@ -97,7 +118,7 @@ function resolve_clue(game, old_game, action, inf_possibilities, focused_card) {
 
 	const correct_match = inf_possibilities.find(p => focused_card.matches(p));
 
-	if (!inference_known(inf_possibilities) && target !== state.ourPlayerIndex && !correct_match.save) {
+	if (correct_match && !inference_known(inf_possibilities) && target !== state.ourPlayerIndex && !correct_match.save) {
```

Comment:
Could you explain why we only need to look for symmetric connections when there is a correct match? It seems a bit strange either way, but I'm curious if you had any thoughts on this.

Reply:
Oh sorry, this change was a quick crash fix that I didn't look too deeply into at the time. I ran into a crash while playing in one game where correct_match was undefined or null and so trying to access correct_match.save crashes, i.e. this usually only runs when we have a correct match because if we didn't it would crash.

I agree it probably makes more sense to find symmetric connections either way, and can change it to avoid using correct match if we don't have one.